packages/langium/src/lsp/node-kind-provider.ts

```diff
@@ -13,19 +13,35 @@ import { CompletionItemKind, SymbolKind } from 'vscode-languageserver';
 export interface NodeKindProvider {
     /**
      * Returns a `SymbolKind` as used by `WorkspaceSymbolProvider` or `DocumentSymbolProvider`.
+     * @param node AST node or node description.
+     * @returns {SymbolKind} - The corresponding symbol kind.
```

Comment:
Detail: the return type information is redundant (already specified on TypeScript level). I'd also remove the leading `-`.

<img width="701" alt="Screenshot 2024-07-17 at 09 25 44" src="https://github.com/user-attachments/assets/b2cf6713-7026-4af3-9c16-627244944878">

Reply:
done - https://github.com/eclipse-langium/langium/pull/1580/commits/7e180e6028b78fbe041a1e42f522fff3b0b2bbe0